Ginger/GingerCoreNET/GingerCoreNET - Backup.csproj

```diff
@@ -0,0 +1,1442 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
```

Comment:
Remove this backup file as it is not needed

Reply:
done